llvm/tools/llvm-link/llvm-link.cpp

```diff
@@ -393,8 +393,16 @@ static bool linkFiles(const char *argv0, LLVMContext &Context, Linker &L,
   // Similar to some flags, internalization doesn't apply to the first file.
   bool InternalizeLinkedSymbols = false;
   for (const auto &File : Files) {
+    auto ErrOrExpected = MemoryBuffer::getFileOrSTDIN(File);
```

Comment:
Nit, this filename is a little weird, maybe `BufferOrErr`?

Reply:
Check! I'll keep that in mind and address along with further potential feedback.
(Sorry, did not see this comment at the time of hitting the button.)